project/Dependencies.scala

```diff
@@ -6,121 +6,97 @@ object Dependencies {
   // WARNING: Please Scala update versions in PluginCross.scala too
   val scala212 = "2.12.10"
   val scala213 = "2.13.1"
-  lazy val checkPluginCross = settingKey[Unit]("Make sure scalaVersion match up")
+  val checkPluginCross = settingKey[Unit]("Make sure scalaVersion match up")
   val baseScalaVersion = scala212
-  def nightlyVersion: Option[String] = sys.props.get("sbt.build.version")
+  def nightlyVersion   = sys.props.get("sbt.build.version")
 
   // sbt modules
   private val ioVersion = nightlyVersion.getOrElse("1.4.0-M2")
-  private val lmVersion =
-    sys.props.get("sbt.build.lm.version") match {
-      case Some(version) => version
-      case _             => nightlyVersion.getOrElse("1.3.0")
-    }
-  val zincVersion = nightlyVersion.getOrElse("1.3.1")
+  private val lmVersion = sys.props.get("sbt.build.lm.version").orElse(nightlyVersion).getOrElse("1.3.0")
+  val zincVersion       = nightlyVersion.getOrElse("1.3.1")
 
   private val sbtIO = "org.scala-sbt" %% "io" % ioVersion
 
   private val libraryManagementCore = "org.scala-sbt" %% "librarymanagement-core" % lmVersion
-  private val libraryManagementIvy = "org.scala-sbt" %% "librarymanagement-ivy" % lmVersion
-
-  val launcherVersion = "1.1.3"
-  val launcherInterface = "org.scala-sbt" % "launcher-interface" % launcherVersion
-  val rawLauncher = "org.scala-sbt" % "launcher" % launcherVersion
-  val testInterface = "org.scala-sbt" % "test-interface" % "1.0"
-  val ipcSocket = "org.scala-sbt.ipcsocket" % "ipcsocket" % "1.0.0"
-
-  private val compilerInterface = "org.scala-sbt" % "compiler-interface" % zincVersion
-  private val compilerClasspath = "org.scala-sbt" %% "zinc-classpath" % zincVersion
-  private val compilerApiInfo = "org.scala-sbt" %% "zinc-apiinfo" % zincVersion
-  private val compilerBridge = "org.scala-sbt" %% "compiler-bridge" % zincVersion
-  private val zinc = "org.scala-sbt" %% "zinc" % zincVersion
-  private val zincCompile = "org.scala-sbt" %% "zinc-compile" % zincVersion
-  private val zincCompileCore = "org.scala-sbt" %% "zinc-compile-core" % zincVersion
-
-  def getSbtModulePath(key: String, name: String) = {
+  private val libraryManagementIvy  = "org.scala-sbt" %% "librarymanagement-ivy"  % lmVersion
+
+  val launcherVersion   = "1.1.3"
```

Comment:
Is this vertically aligned? I think we use Scalafmt to format these files and I don't think this would stick.

Reply:
💩